src/items/ItemsDetailsPanel.component.tsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import {
+  Box,
+  Collapse,
+  Grid,
+  Link as MuiLink,
+  Tab,
+  Tabs,
+  Typography,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import {
+  CatalogueCategory,
+  CatalogueItem,
+  Item,
+  UsageStatusType,
+} from '../app.types';
+import { useManufacturer } from '../api/manufacturer';
+
+function TabPanel(props: any) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+      {...other}
+    >
+      <Collapse in={value === index}>
+        <Grid container spacing={2}>
+          {children}
+        </Grid>
+      </Collapse>
+    </div>
+  );
+}
+
+export interface ItemsDetailsPanelProps {
+  itemData: Item;
+  catalogueItemIdData: CatalogueItem;
+  catalogueCategoryData: CatalogueCategory;
+}
+
+function ItemsDetailsPanel(props: ItemsDetailsPanelProps) {
+  const { catalogueItemIdData, catalogueCategoryData, itemData } = props;
+  const [tabValue, setTabValue] = React.useState(0);
+  const { data: manufacturerData } = useManufacturer(
+    catalogueItemIdData.manufacturer_id
+  );
+  const handleTabChange = (event: React.SyntheticEvent, newValue: number) => {
+    setTabValue(newValue);
+  };
+  return (
+    <Grid
+      container
+      spacing={0}
+      flexDirection="column"
+      // Stop any further propagation to prevent a table select from being triggered
+      // by clicks inside this grid
+      onClick={(e) => e.stopPropagation()}
+    >
+      <Grid item sx={{ mb: 4 }} xs={12}>
+        <Tabs value={tabValue} onChange={handleTabChange}>
+          <Tab label="Details" />
+          <Tab label="Properties" />
+          <Tab label="Manufacturer" />
+          <Tab label="Notes" />
+        </Tabs>
+      </Grid>
+      <Grid item container sx={{ ml: 2 }} xs={12}>
+        <TabPanel value={tabValue} index={0}>
+          <Grid item container spacing={0}>
+            <Grid item xs={12}>
+              <Typography variant="h4">{catalogueItemIdData.name}</Typography>
+              <Typography sx={{ my: 1 }} variant="h6">
+                Description:
+              </Typography>
+              <Typography sx={{ mb: 1 }} variant="body1">
```

Comment:
```suggestion
              <Typography sx={{ mb: 1 }} variant="body1" color="text.secondary">
```
To match the others

Reply:
Good spot, this would need to be change in the catalogue items landing page and the manufacturers aswell